Add host resource (#6)
ansible/terraform-provider-aap: README.md

Then you can run acceptance tests with `make testacc`.

> Acceptance tests for job, host and group resources require you to export the following environment variable:

Reviewer: ```suggestion
Following environment variables must be exported in order to run acceptance tests for job, host and group resources:
```
